openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/SchemaTablesTab.tsx

```diff
@@ -69,6 +75,46 @@ function SchemaTablesTab({
   pagingInfo,
 }: Readonly<SchemaTablesTabProps>) {
   const { t } = useTranslation();
+  const [localTableData, setLocalTableData] = useState<Table[]>([]);
+
+  const { permissions } = usePermissionProvider();
+
+  const allowEditDisplayNamePermission = useMemo(() => {
+    return (
+      !isVersionView &&
+      (permissions.table.EditAll || permissions.table.EditDisplayName)
+    );
+  }, [permissions, isVersionView]);
+
+  const handleDisplayNameUpdate = useCallback(
+    async (data: EntityName, id?: string) => {
+      try {
+        const tableDetails = localTableData.find((table) => table.id === id);
+        if (!tableDetails) {
+          throw new Error(t('error.tableNotFound'));
```

Comment:
same here around the error object in translation file.

Reply:
Corrected it, removed `throw new Error` and used `return` instead.